dev-scheme/racket/files/Makefile-nostrip.patch

```diff
@@ -0,0 +1,23 @@
+index 3393e0c55e..6a1c3e150b 100644
+--- a/bc/Makefile.in
++++ b/bc/Makefile.in
+@@ -505,8 +505,6 @@ unix-install-libs-cgc:
+ 	cd ..; $(ICP_LIB) bc/libmzgc.@LIBSFX@ "$(DESTDIR)$(libdir)/libmzgc.@LIBSFX@"
+ 	cd ..; $(ICP_LIB) bc/libracket.@LIBSFX@ "$(DESTDIR)$(libdir)/libracket.@LIBSFX@"
+ 	cd ..; $(ICP_STATIC_LIB) bc/rktio/librktio.@LIBSFX@ "$(DESTDIR)$(libdir)/librktio.@LIBSFX@"
+-	cd ..; $(STRIP_LIB_DEBUG) "$(DESTDIR)$(libdir)/libmzgc.@LIBSFX@"
+-	cd ..; $(STRIP_LIB_DEBUG) "$(DESTDIR)$(libdir)/libracket.@LIBSFX@"
+ 
+ unix-no-install-libs-cgc:
+ 	$(NOOP)
+@@ -524,9 +522,8 @@ unix-install-3m:
+ 	$(RESTORE_SIGNATURE) "$(DESTDIR)$(bindir)/racket@MMM_INSTALLED@"
+ 
+ unix-install-libs-3m:
+-	cd ..; $(ICP_LIB) bc/libracket3m.@LIBSFX@ "$(DESTDIR)$(libdir)/libracket3m.@LIBSFX@"
++	cd ..; libtool --mode=install install bc/libracket3m.@LIBSFX@ "$(DESTDIR)$(libdir)/libracket3m.@LIBSFX@"
```

Comment:
We probably don't want to call libtool directly, but instead use the "bundled" / generated one. That might be $(ICP_LIB) in this case (not sure).

Is there a reason you needed to drop it?

Reply:
> We probably don't want to call libtool directly, but instead use the "bundled" / generated one. That might be $(ICP_LIB) in this case (not sure).
> 
> Is there a reason you needed to call directly?

ICP_LIB was a call to libtool that included command to strip the library

in logs I saw this
```log
/var/tmp/portage/dev-scheme/racket-8.2/work/racket-8.2/src/bc/rktio/../lt/libtool --mode=link --tag=CC x86_64-pc-li
nux-gnu-gcc -o  librktio.la rktio_fs.lo rktio_fd.lo rktio_file.lo rktio_poll_set.lo rktio_ltps.lo rktio_sleep.lo rk
tio_network.lo rktio_pipe.lo rktio_process.lo rktio_signal.lo rktio_envvars.lo rktio_fs_change.lo rktio_flock.lo rk
tio_shellex.lo rktio_time.lo rktio_cpu.lo rktio_syslog.lo rktio_convert.lo rktio_sha1.lo rktio_sha2.lo rktio_dll.lo
 rktio_error.lo rktio_hash.lo rktio_wide.lo rktio_console.lo rktio_main.lo

 ...

 libtool --mode=compile --tag=CC x86_64-pc-linux-gnu-gcc -march=znver1 -O3 -falign-functions=32 -fno-common -fstack-protector-strong -pipe -fdiagnostics-color=always -frecord-gcc-switches  -Wall   -I./.. -I./../include -I./../../version -I./../../rktio -I../rktio -pthread -I/usr/lib64/libffi/include   -DUSE_SENORA_GC  -DMZ_USES_SHARED_LIB -c ./salloc.c -o salloc.lo
 ```
 which means it sometimes calls bundled libtool and sometimes it doesn't
 
 
 In case of libracket3m it wasn't calling the bundled one:
 ```log
 make unix-install-libs-3m
make[5]: Entering directory '/var/tmp/portage/dev-scheme/racket-8.2/work/racket-8.2/src/bc'
cd ..; libtool --mode=install install -s bc/libracket3m.la "/var/tmp/portage/dev-scheme/racket-8.2/image/usr/lib64/
libracket3m.la"
libtool: install: install bc/.libs/libracket3m-8.2.so /var/tmp/portage/dev-scheme/racket-8.2/image/usr/lib64/librac
ket3m-8.2.so
libtool: install: x86_64-pc-linux-gnu-strip --strip-unneeded /var/tmp/portage/dev-scheme/racket-8.2/image/usr/lib64
/libracket3m-8.2.so
libtool: install: (cd /var/tmp/portage/dev-scheme/racket-8.2/image/usr/lib64 && { ln -s -f libracket3m-8.2.so libra
cket3m.so || { rm -f libracket3m.so && ln -s libracket3m-8.2.so libracket3m.so; }; })
libtool: install: install bc/.libs/libracket3m.lai /var/tmp/portage/dev-scheme/racket-8.2/image/usr/lib64/libracket
3m.la
libtool: install: install bc/.libs/libracket3m.a /var/tmp/portage/dev-scheme/racket-8.2/image/usr/lib64/libracket3m
.a
libtool: install: x86_64-pc-linux-gnu-strip --strip-debug /var/tmp/portage/dev-scheme/racket-8.2/image/usr/lib64/li
bracket3m.a
libtool: install: chmod 644 /var/tmp/portage/dev-scheme/racket-8.2/image/usr/lib64/libracket3m.a
libtool: install: x86_64-pc-linux-gnu-ranlib /var/tmp/portage/dev-scheme/racket-8.2/image/usr/lib64/libracket3m.a
libtool: warning: remember to run 'libtool --finish /usr/lib64'
```

Reply:
That's a bit annoying, but it's not your job to necessarily fix this.

That said, I'd prefer to keep $(ICP_LIB) and just drop whereever the strip is coming from, but if we can't do that, so be it.

Reply:
> That said, I'd prefer to keep $(ICP_LIB) and just drop whereever the strip is coming from, but if we can't do that, so be it.

Those are the changes it took to fix it the way you want: https://github.com/racket/racket/commit/92d202060201f569fafc348c4f64a2965b55287f

I opted to have a simpler patch.

Reply:
Works for me for now! Thank you!

Reply:
Looks good..